core/io/http_client_tcp.cpp

```diff
@@ -788,4 +788,4 @@ HTTPClientTCP::HTTPClientTCP() {
 
 HTTPClient *(*HTTPClient::_create)() = HTTPClientTCP::_create_func;
 
-#endif // #ifndef JAVASCRIPT_ENABLED
+#endif // #ifndef WEB_ENABLED
```

Comment:
Nitpick but usually we'd just do:
```suggestion
#endif // WEB_ENABLED
```
Doesn't convey the `n` part of the `#ifndef` but it doesn't matter too much, consistency is easier this way.